appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java

```diff
@@ -132,12 +132,15 @@ public YoungAndroidProjectService(StorageIo storageIo) {
    * Returns project settings that can be used when creating a new project.
    */
   public static String getProjectSettings(String icon, String vCode, String vName,
-    String useslocation, String aName, String sizing, String showListsAsJson, String tutorialURL, String subsetJSON,
-    String actionBar, String theme, String primaryColor, String primaryColorDark, String accentColor) {
+    String useslocation, String usescamera, String usesmicrophone, String aName, String sizing, String showListsAsJson,
```

Comment:
The old code was wrong w.r.t. indentation here, but you should go ahead and fix it (should be 6 spaces from beginning of line).